requirements_test.txt

```diff
@@ -42,3 +42,6 @@ scikit-optimize
 zoopt
 
 s3fs>=2022.8.2
+
+# required for explain tests
+captum
```

Comment:
Should you just install the `requirements_explain.txt` deps for the LLM test suite?

Reply:
Done